database/insertData/01_template_C_userRegistration.js

```diff
@@ -16,12 +16,14 @@ exports.queries = [
           templateSectionsUsingId: {
             create: [
               {
+                id: 1004
```

Comment:
```suggestion
                id: 1002
```

Just because previous ones are 1000 and 1001

Reply:
No, the previous **Section** IDs (in Template B) are 1002, 1003 -- perhaps you got mixed up with the Template or Element IDs? After fresh init, you can see the sequential IDs. :)  🤷‍♂️

<img width="528" alt="Screen Shot 2021-01-29 at 11 35 34 PM" src="https://user-images.githubusercontent.com/5456533/106264575-b369db00-628a-11eb-9296-e35da67b042a.png">

Reply:
Yes, sorry about my confusion : D